test/integration_test.js

```diff
@@ -18,7 +18,10 @@ const TEST_KINTO_SERVER = "http://0.0.0.0:8888/v1";
 describe("Integration tests", () => {
   let sandbox, server, api;
 
-  before(() => server = new KintoServer(TEST_KINTO_SERVER));
+  before(function() {
+    this.timeout(20000);
```

Comment:
I didn't open any ticket on this. But @n1k0 said that the last versions takes a lot longer to start